generators/app/gulpfile.js

```diff
@@ -98,6 +98,9 @@ function build() {
         // load them.
         buildStream = buildStream.pipe(polymerProject.bundler());
 
+        // Okay, now let's generate the HTTP/2 Push Manifest
```

Comment:
Having "Okay," twice feels oddly robotic, removing it and starting with "Now let's ..." should be fine